tests/postgres_tests/test_constraints.py

```diff
@@ -266,6 +266,53 @@ def test_opclass_func(self):
         self.assertNotIn(constraint.name, self.get_constraints(Scene._meta.db_table))
         Scene.objects.create(scene="ScEnE 10", setting="Sir Bedemir's Castle")
 
+    @isolate_apps("postgres_tests")
+    def test_opclass_func_validate_constraints(self):
+        """
+        Regression test for #35275
+
+        Ensures that `OpClass` is correctly unwrapped when validating `UniqueConstraint`
+        using `OpClass`, otherwise it leads to an invalid SQL query that raises a syntax
+        error. The test also ensures that we still apply the operation used in `OpClass`
+        when validating the constraints (in this test, `Lower`).
+        """
```

Comment:
We generally don't reference ticket number in test docstrings. Also, to add more explanation, docstrings should state the expected behavior and omit prefixes like "Ensure that" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Personally, I'd drop this docstring it's unnecessary.